<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1172</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>1051</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="wgInicial">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1091</width>
        <height>571</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>1088</width>
        <height>571</height>
       </size>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLabel" name="labelMensaje">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>90</y>
        <width>311</width>
        <height>171</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>8514oem</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="accessibleIdentifier">
       <string notr="true"/>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    border-image: url(&quot;C:/Users/Dell/Documents/Documentos/PROYECTO/Gamificacion/images/com2.png&quot;) 0 0 0 0 stretch stretch;
    font-family: &quot;8514oem&quot;;      /* Cambia Arial por la fuente que quieras */
    font-size: 12px;           /* Tamaño del texto en píxeles */
    color: black;              /* Color del texto */
 qproperty-alignment: 'AlignCenter';  /* Centrar texto */
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="wgDentro">
     <widget class="QGraphicsView" name="graphicsViewDentro">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1051</width>
        <height>581</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="giro">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>280</y>
        <width>181</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>INICIAR GIRO</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="wgMapa">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1051</width>
        <height>591</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    border-image: url(&quot;C:/Users/Dell/Documents/Documentos/PROYECTO/Gamificacion/images/descarga (1).jpg&quot;) 0 0 0 0 stretch stretch;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="wgEntrada">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1051</width>
        <height>581</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    border-image: url(&quot;C:/Users/Dell/Documents/Documentos/PROYECTO/Gamificacion/images/fon2.jpg&quot;) 0 0 0 0 stretch stretch;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="bnNuevaPartida">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>290</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>8514oem</family>
        <pointsize>9</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-image: url(&quot;C:/Users/Dell/Documents/Documentos/PROYECTO/Gamificacion/images/bnplay.png&quot;) 0 0 0 0 stretch stretch;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="bnNuevaPartida_3">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>340</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>8514oem</family>
        <pointsize>9</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-image: url(&quot;C:/Users/Dell/Documents/Documentos/PROYECTO/Gamificacion/images/bnexit.png&quot;) 0 0 0 0 stretch stretch;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="bnNuevaPartida_4">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>220</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>8514oem</family>
        <pointsize>9</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-image: url(&quot;C:/Users/Dell/Documents/Documentos/PROYECTO/Gamificacion/images/menu.png&quot;) 0 0 0 0 stretch stretch;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1172</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
